src/typescript/services/apiService.ts

```diff
@@ -0,0 +1,24 @@
+class ApiService {
+    private baseUrl: string;
+    private path: string;
+
+    constructor(baseUrl: string, path: string) {
+        this.baseUrl = baseUrl;
+        this.path = path;
+    }
+
+    async get(): Promise<any> {
+        try {
+            const response = await fetch(`${this.baseUrl}${this.path}?_expand=category`);
```

Comment:
If this is common way to get every data. Should we use expand category here?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!